src/calchartdoc.cpp

```diff
@@ -279,6 +281,25 @@ wxInputStream& CalChartDoc::LoadObject(wxInputStream& stream)
 }
 #endif
 
+bool CalChartDoc::exportViewerFile(const wxString& filepath) {    
+    JSONElement mainObjectElement = JSONElement::makeObject();
+    
+    JSONDataObjectAccessor mainObject = mainObjectElement;
```

Comment:
You can't do `JSONDataObjectAccessor mainObject = JSONElement::makeObject()`? Dpes that kind of polymorphism not work in C++?